packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.styles.ts

```diff
@@ -10,7 +10,7 @@ import { IActivityItemStyles } from './ActivityItem.Props';
 
 const DEFAULT_PERSONA_SIZE = '32px';
 const COMPACT_PERSONA_SIZE = '16px';
-const DEFAULT_ICON_SIZE = '24px';
+const DEFAULT_ICON_SIZE = '16px';
```

Comment:
no change file. Also, does this change layout?

Reply:
Just added a rush change. Flexbox gracefully handles the smaller icon size, so the layout doesn't change.